mlrun/launcher/factory.py

```diff
@@ -11,18 +11,25 @@
 # WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
 # See the License for the specific language governing permissions and
 # limitations under the License.
+import mlrun.config
 import mlrun.errors
-from mlrun.launcher import BaseLauncher, ClientLocalLauncher, ClientRemoteLauncher
+import mlrun.launcher.base
+import mlrun.launcher.local
+import mlrun.launcher.remote
 
 
 class LauncherFactory(object):
     @staticmethod
-    def create_launcher(local: bool = False) -> BaseLauncher:
+    def create_launcher(
+        is_remote, local: bool = False
+    ) -> mlrun.launcher.base.BaseLauncher:
```

Comment:
```suggestion
    def create_launcher(
        is_remote: bool, local: bool = False
    ) -> mlrun.launcher.base.BaseLauncher:
```
also missing description for the params in the docstring